### PR TITLE
Fix frontend lint warnings

### PR DESCRIPTION
**Pull request by connor-bechthold**
## Implementation description
Fixed frontend lint warnings because they were starting to bloat PR's:
Screenshot 2023-06-20 at 1 59 29 AM

## Steps to test
Nothing specific, just running through the app and ensuring it doesn't break anywhere should be good (testing isn't really needed)

## What should reviewers focus on?
* 

## Checklist
- [ ] My PR name is descriptive and in imperative tense
- [ ] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [ ] I have run the appropriate linter(s)
- [ ] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR
- [ ] If I have made API changes, I have updated the [REST API Docs](https://www.notion.so/uwblueprintexecs/REST-Endpoints-05ce60312bb943439dfda42bb1318536)
- [ ] IF I have made changes to the db/models, I have updated the [Data Models Page](https://www.notion.so/uwblueprintexecs/Data-Models-760f8aa06b244eb0842c079ad77987b0)
- [ ] I have updated other Docs as needed

### CODE REVIEW

**Comment by safchow** [comment]
noticing I made an oopsie in one of my previous PRs:
```
typescript@^4.1.2:
  version "4.1.2"
  resolved "https://registry.yarnpkg.com/typescript/-/typescript-4.9.5.tgz#095979f9bcc0d09da324d58d03ce8f8374cbe65a"
  integrity sha512-1FXk9E2Hm+QzZQ7z+McJiHL4NW1F2EzMu9Nq9i3zAaGqibafqYwCVU6WyWAuyQRRzOlxou8xZSyXLEN8oKj24g==
```

Notice how we use 4.1.2, but the version coming from the URL is 4.9.5.

**Comment by connor-bechthold** [comment]
> 



> 

Yeah it should be version: 4.9.5

**Comment by connor-bechthold** [comment]
> noticing I made an oopsie in one of my previous PRs:
> 
> ```
> typescript@^4.1.2:
>   version "4.1.2"
>   resolved "https://registry.yarnpkg.com/typescript/-/typescript-4.9.5.tgz#095979f9bcc0d09da324d58d03ce8f8374cbe65a"
>   integrity sha512-1FXk9E2Hm+QzZQ7z+McJiHL4NW1F2EzMu9Nq9i3zAaGqibafqYwCVU6WyWAuyQRRzOlxou8xZSyXLEN8oKj24g==
> ```
> 
> Notice how we use 4.1.2, but the version coming from the URL is 4.9.5.

Yeah it should be `version: 4.9.5`. If the goal is to be using `4.1.2`, the caret needs to be removed.

**Comment by safchow** [comment]
> > noticing I made an oopsie in one of my previous PRs:
> > ```
> > typescript@^4.1.2:
> >   version "4.1.2"
> >   resolved "https://registry.yarnpkg.com/typescript/-/typescript-4.9.5.tgz#095979f9bcc0d09da324d58d03ce8f8374cbe65a"
> >   integrity sha512-1FXk9E2Hm+QzZQ7z+McJiHL4NW1F2EzMu9Nq9i3zAaGqibafqYwCVU6WyWAuyQRRzOlxou8xZSyXLEN8oKj24g==
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Notice how we use 4.1.2, but the version coming from the URL is 4.9.5.
> 
> Yeah it should be `version: 4.9.5`. If the goal is to be using `4.1.2`, the caret needs to be removed.

lets stick to 4.1.2, would you be able to update the yarn.lock file in this pr?

After the update, run the linter again

**Comment by connor-bechthold** [comment]
> > > noticing I made an oopsie in one of my previous PRs:
> > > ```
> > > typescript@^4.1.2:
> > >   version "4.1.2"
> > >   resolved "https://registry.yarnpkg.com/typescript/-/typescript-4.9.5.tgz#095979f9bcc0d09da324d58d03ce8f8374cbe65a"
> > >   integrity sha512-1FXk9E2Hm+QzZQ7z+McJiHL4NW1F2EzMu9Nq9i3zAaGqibafqYwCVU6WyWAuyQRRzOlxou8xZSyXLEN8oKj24g==
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > Notice how we use 4.1.2, but the version coming from the URL is 4.9.5.
> > 
> > 
> > Yeah it should be `version: 4.9.5`. If the goal is to be using `4.1.2`, the caret needs to be removed.
> 
> lets stick to 4.1.2, would you be able to update the yarn.lock file in this pr?
> 
> After the update, run the linter again

Doing this breaks `yarn install`. I also tried clearing cache
`#0 2.011 error https://registry.npmjs.org/object-assign/-/object-assign-4.1.1.tgz: Integrity check failed for "object-assign" (computed integrity doesn't match our records, got "sha512-rJgTQnkUnH1sFw8yT6VSU3zD3sWmu6sZhIseY8VX+GRu3P6F7Fu+JNDoXfklElbLJSnc3FUQHVe4cU5hj+BcUg== sha1-IQmtx5ZYh8/AXLvUQsrIv7s2CGM=")`

Can we just leave the version as is? Pretty sure it's best practice to always use the latest stable version anyways

**Comment by safchow** [comment]
> > > > noticing I made an oopsie in one of my previous PRs:
> > > > ```
> > > > typescript@^4.1.2:
> > > >   version "4.1.2"
> > > >   resolved "https://registry.yarnpkg.com/typescript/-/typescript-4.9.5.tgz#095979f9bcc0d09da324d58d03ce8f8374cbe65a"
> > > >   integrity sha512-1FXk9E2Hm+QzZQ7z+McJiHL4NW1F2EzMu9Nq9i3zAaGqibafqYwCVU6WyWAuyQRRzOlxou8xZSyXLEN8oKj24g==
> > > > ```
> > > > 
> > > > 
> > > >     
> > > >       
> > > >     
> > > > 
> > > >       
> > > >     
> > > > 
> > > >     
> > > >   
> > > > Notice how we use 4.1.2, but the version coming from the URL is 4.9.5.
> > > 
> > > 
> > > Yeah it should be `version: 4.9.5`. If the goal is to be using `4.1.2`, the caret needs to be removed.
> > 
> > 
> > lets stick to 4.1.2, would you be able to update the yarn.lock file in this pr?
> > After the update, run the linter again
> 
> Doing this breaks `yarn install`. I also tried clearing cache `#0 2.011 error https://registry.npmjs.org/object-assign/-/object-assign-4.1.1.tgz: Integrity check failed for "object-assign" (computed integrity doesn't match our records, got "sha512-rJgTQnkUnH1sFw8yT6VSU3zD3sWmu6sZhIseY8VX+GRu3P6F7Fu+JNDoXfklElbLJSnc3FUQHVe4cU5hj+BcUg== sha1-IQmtx5ZYh8/AXLvUQsrIv7s2CGM=")`
> 
> Can we just leave the version as is? Pretty sure it's best practice to always use the latest stable version anyways

merged a fix in earlier today #121

File: frontend/src/APIClients/EntityAPIClient.ts [hunk 1]
```diff
@@ -41,6 +41,7 @@ const create = async ({
       headers: { Authorization: bearerToken },
     });
     return data;
+    /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
```
Comment (connor-bechthold) [inline frontend/src/APIClients/EntityAPIClient.ts:44]: These are temporary as we don't use this file anyways

**Pull request closed without merging**